hs_core/models.py

```diff
@@ -1024,6 +1025,56 @@ def update(cls, element_id, **kwargs):
     def remove(cls, element_id):
         raise ValidationError("Coverage element can't be deleted.")
 
+    def add_to_xml_container(self, container):
```

Comment:
This is a serializer in disguise. At some point we need to clean up the serializers and their patterns. This will do for now.